### PR TITLE
Adding `updateContext` method to allow child addin to receive updates from parent

### PR DESCRIPTION
**Pull request by Blackbaud-ScottZetrouer**
Child addin will be able to receive updated context information from the parent host.

### CODE REVIEW

File: src/addin/client-interfaces/addin-client-callbacks.ts [hunk 1]
```diff
@@ -15,6 +15,11 @@ export interface AddinClientCallbacks {
    */
   buttonClick?: () => void;
 
+  /**
+   * Callback raised indicating context was updated by the host.
+   */
+  updateContext?: () => void;
```
Comment (michael-tims) [inline src/addin/client-interfaces/addin-client-callbacks.ts:21]: Should take a `context: any` param I believe.

File: src/addin/addin-client.ts [hunk 2]
```diff
@@ -406,6 +406,11 @@ export class AddinClient {
               this.args.callbacks.buttonClick();
             }
             break;
+          case 'update-context':
+            if (this.args.callbacks.updateContext) {
+              this.args.callbacks.updateContext();
```
Comment (michael-tims) [inline src/addin/addin-client.ts:411]: And then pass in `data.message` to updateContext. `this.args.callbacks.updateContext(data.message);`

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,7 @@
+# 1.0.17 (2020-05-06)
+- Added `updateContext` method to allow child addin to receive updated context information from parent.
```
Comment (michael-tims) [inline CHANGELOG.md:2]: Rephrase to be "Added `updateContext` callback to notify add-in when context information has been updated."

File: src/addin/addin-client.ts (hunk 2 above)
Comment (Blackbaud-ScottZetrouer) [reply src/addin/addin-client.ts]: resolved

File: CHANGELOG.md (hunk 3 above)
Comment (Blackbaud-ScottZetrouer) [reply CHANGELOG.md]: should be fixed in latest

**Comment by Blackbaud-ScottZetrouer** [comment]
should have addressed your comments in latest commit.

**Pull request merged**